README.md

```diff
@@ -40,18 +38,26 @@ By default you get the following:
 ## Usage
 
 In order to use this module, classify the node running PE PostgreSQL with the `pe_databases` class.
-That node is the Primary Master in a Monolithic installation, or the PE PuppetDB host in a Split install.  
+That node is the Primary Server in most instances, but there may be one or more Servers with this role in an XL deployment  
 
 To classify via the PE Console, create a new node group called "PE Database Maintenance".
```

Comment:
This note doesn't even say to add the class.

Reply:
`In order to use this module, classify the node running PE PostgreSQL with the `pe_databases` class.` is that not sufficient? should we put in a hiera reference or something

Reply:
Yeah. I just saw that this whole note says to create a node group and pin the node, but never says to add the class to the node group. However, I think we should just remove this section with the idea of auto inclusion.